### PR TITLE
Add tests for comments counter function

### PR DESCRIPTION
**Pull request by amon-cofie**
### Changes:

- Added testing for comments counter function

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -12,22 +12,3 @@ document.addEventListener('DOMContentLoaded', () => {
...
-//   const response = request.text();
-//   return response;
-// };
-
-// addGame(
-//   'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/',
-//   {
-//     name: 'myNewApp',
-//   }
-// ).then((response) => console.log(response));
```
Comment (juaryimami) [inline src/index.js:33]: - Its better to remove commented line of old code for better code standard.

File: src/modules/comment-pop-up-structure.js [hunk 2]
```diff
@@ -86,7 +86,7 @@ const commentPopUp = (obj, id, getCommentCallback, setCommentCallback) => {
     (data) => data,
     (error) => {
       throw new Error(error);
-    }
+    },
   );
```
Comment (juaryimami) [inline src/modules/comment-pop-up-structure.js:91]: - try to modularize your codes for better code maintanablity.

**Review by juaryimami — APPROVED** [review]
## Project Approved 🙂👍 

Hi, @amon-cofie , This is @juaryimami your colleague.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional `readme`

DiCaprion cheers

## Optional suggestions:

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this issue. However, I strongly recommend you take them into account as they can make your code better._

- NAN

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the issue thread if something is not 100% clear.
Please, remember to tag me [@juaryimami ] in your question so I can receive the notification.

------

File: src/index.js (hunk 1 above)
Comment (amon-cofie) [reply src/index.js:33]: Oh thanks for pointing that out. I totally forgot to do that :+1:

File: src/modules/comment-pop-up-structure.js (hunk 2 above)
Comment (amon-cofie) [reply src/modules/comment-pop-up-structure.js:91]: thanks

**Pull request merged**